docs/index.md

```diff
@@ -44,6 +43,10 @@
 * [Creating a Multidomain Design](./cookbook/creating-a-multidomain-design.md)
 * [Working with Multiple Cron Instances](./cookbook/working-with-multiple-cron-instances.md)
 
+## Model
```

Comment:
I think more articles should be moved to this sections - especially those regarding Entities because those are part of model. Or Not?